src/staging/gatsby-node.js

```diff
@@ -308,26 +307,39 @@ const staticQueryPath = `
   }
 }`
 
-async function createRedirects({ graphql, actions, result }) {
+async function createRedirects({ graphql, actions, reporter }) {
   const response = await strapiApiRequest(graphql, staticQueryPath)
   const resultPark = await strapiApiRequest(graphql, parkQuery)
 
   const redirects = response.data.allStrapiLegacyRedirect.nodes
   const parks = resultPark.data.allStrapiProtectedArea.nodes
-  
+
   redirects.map(item => {
     return actions.createRedirect({
       fromPath: item.fromPath,
       toPath: item.toPath,
     })
   })
   parks.map(park => {
-    const oldUrl = parseUrl(park.oldUrl);
-    if(oldUrl?.pathname !== `/${park.slug}/`) {
-      return actions.createRedirect({
-        fromPath: oldUrl.pathname,
-        toPath: park.slug,
-      })
+    let oldUrl = '';
+    try {
+      oldUrl = parseUrl(park.oldUrl);
+      if (oldUrl?.pathname !== `/${park.slug}/`) {
+        return actions.createRedirect({
+          fromPath: oldUrl.pathname,
+          toPath: park.slug,
+        })
+      }
+    } catch (error) {
+      reporter.warn(`Field oldUrl for park ${park.id} not found, 
+      error message: ${JSON.stringify(error)}
```

Comment:
1. Can you change the message to say **"Field oldUrl for park ${park.id} could not be parsed"**
2. We probably don't need to print the error object.  It's always the same, and there is enough information in the "park" object for anyone to track down the cause.  
3. Please remove the `===========`